support/commercial.rst

```diff
@@ -0,0 +1,12 @@
+.. class:: hide-rst-heading
+
+Paid Support Services
```

Comment:
Maybe rather "Paid Support and Services"

e.g. rsync.net could be also here, it is service-only (backup storage provider), no support.